package.json

```diff
@@ -22,7 +22,7 @@
     "rewrite"
   ],
   "dependencies": {
-    "detective": "~4.1.0",
+    "detective": "4.x.x",
```

Comment:
Thanks! Please just change the `~` to a `^`, I prefer not to use `x` in package versions.